docs/streams/developer-guide/dsl-api.html

```diff
@@ -3393,6 +3393,10 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
 				      but this simple example creates a buffer with no
 				      upper bound.
 				    </dd>
+                    <dt><code>withLoggingDisabled()</code></dt>
+                    <dd>
+                      This configures the suppression operator to disable logging for changelog entries.
+                    </dd>
```

Comment:
Not sure if we want to add this in this example? \cc @vvcephei WDYT?

Reply:
Thanks @mjsax , I agree; this is an advanced configuration that shouldn't be used most of the time, so I wouldn't put it in the example.

Reply:
update code thanks!